pkg/postage/batchstore/store.go

```diff
@@ -5,62 +5,142 @@
 package batchstore
 
 import (
+	"errors"
 	"math/big"
 
 	"github.com/ethersphere/bee/pkg/postage"
 	"github.com/ethersphere/bee/pkg/storage"
 )
 
 const (
-	batchKeyPrefix = "batchKeyPrefix"
-	stateKey       = "stateKey"
+	batchKeyPrefix  = "batch"
```

Comment:
very generic key names that might conflict with others. can you add a bit more text here?